python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1853,6 +1857,107 @@ def get_event_records(
+
+        Args:
+            asset_records_filter (Optional[AssetRecordsFilter]): the filter by which to filter event
+                records.
+            limit (Optional[int]): Number of results to get. Defaults to infinite.
+            ascending (Optional[bool]): Sort the result in ascending order if True, descending
+                otherwise. Defaults to descending.
+
+        Returns:
+            List[EventLogRecord]: List of event log records stored in the event log storage.
+        """
+        return self._event_storage.get_materialization_records(
+            asset_key, asset_records_filter, limit, ascending
+        )
+
+    @public
+    @traced
+    def get_planned_materialization_records(
+        self,
+        asset_key: Optional[AssetKey] = None,
+        asset_records_filter: Optional["AssetRecordsFilter"] = None,
+        limit: Optional[int] = None,
+        ascending: bool = False,
+    ) -> Sequence["EventLogRecord"]:
+        """Return a list of planned materialization records stored in the event log storage.
+
+        Args:
+            asset_records_filter (Optional[AssetRecordsFilter]): the filter by which to filter event
+                records.
+            limit (Optional[int]): Number of results to get. Defaults to infinite.
+            ascending (Optional[bool]): Sort the result in ascending order if True, descending
+                otherwise. Defaults to descending.
+
+        Returns:
+            List[EventLogRecord]: List of event log records stored in the event log storage.
+        """
+        return self._event_storage.get_planned_materialization_records(
+            asset_key, asset_records_filter, limit, ascending
+        )
+
+    @public
+    @traced
+    def get_observation_records(
+        self,
+        asset_key: Optional[AssetKey] = None,
+        asset_records_filter: Optional["AssetRecordsFilter"] = None,
+        limit: Optional[int] = None,
+        ascending: bool = False,
+    ) -> Sequence["EventLogRecord"]:
+        """Return a list of observation records stored in the event log storage.
+
+        Args:
+            asset_records_filter (Optional[AssetRecordsFilter]): the filter by which to filter event
+                records.
+            limit (Optional[int]): Number of results to get. Defaults to infinite.
+            ascending (Optional[bool]): Sort the result in ascending order if True, descending
+                otherwise. Defaults to descending.
+
+        Returns:
+            List[EventLogRecord]: List of event log records stored in the event log storage.
+        """
+        return self._event_storage.get_observation_records(
+            asset_key, asset_records_filter, limit, ascending
+        )
+
+    @public
+    @traced
+    def get_run_status_event_records(
+        self,
+        event_type: "DagsterEventType",
+        filters: Optional["RunStatusEventRecordsFilter"] = None,
+        limit: Optional[int] = None,
+        ascending: bool = False,
```

Comment:
- Seems like `event_type` should be part of the filter.
- `filters` should be singular both for consistency with above and because plural implies a list of `RunStatusEventRecordsFilter`